src/main/java/com/esri/samples/map/create_and_save_map/README.md

```diff
@@ -0,0 +1,52 @@
+<h1>Create and Save Map</h1>
+
+<p>How to create and save a map to your own portal.</p>
+
+<p><img src="CreateAndSaveMap.png"/></p>
+
+<h2>How to use the sample</h2>
+
+Fill in your portal and registered app credentials of the starting dialog to authenticate. Then choose 
+the basemap and layers for your new map. To save the map, choose a title, tags and description (optional), and a folder 
+on your portal. Click the Save button to save the map to the chosen folder.
+
+<h2>How it works</h2>
+
+<p>To create and save a map to your portal for use in an app:</p>
+<ol>
+  <li>Create an <code>ArcGISMap</code> with a <code>Basemap</code> and operational layers</li>
+  <li>Create an <code>OAuthConfiguration</code> with your portal and app credential.</li>
+  <li>Add the configuration to the <code>AuthenticationMangager</code></li>
+  <li>Create a <code>Portal</code> and load it. Use a custom <code>AuthenticationChallengeHandler</code> to 
+  authenticate with your username and password</li>
+  <li>Once authenticated, save the map by calling <code>map.saveMapAsAsync()</code>, passing in the title, tags, 
+  description, and portal folder</li>
+</ol>
+
+<h2>Authentication</h2>
```

Comment:
Might also need to say something about adding a folder to their `My Content` section. If they don't have any folders they will not be able to save anything on the application.
![content](https://cloud.githubusercontent.com/assets/14575538/21400777/a2036bb8-c7a9-11e6-913f-674c7ff76aa9.PNG)